editor/plugins/tiles/tile_map_editor.cpp

```diff
@@ -4000,7 +3937,7 @@ TileMapEditor::TileMapEditor() {
 
 	// TabBar.
 	tabs_bar = memnew(TabBar);
-	tabs_bar->set_tab_alignment(TabBar::ALIGNMENT_CENTER);
+	tabs_bar->set_v_size_flags(SIZE_SHRINK_END);
```

Comment:
Is this even needed? 🤔

Reply:
It was before because it prevents the tabs from expanding vertically when the scroll bar is visible, but not anymore, because I'm removing the ScrollContainer and converting it to use OptionButton overrun behavior. So it can be removed.